README_FORMAT.md

```diff
@@ -0,0 +1,48 @@
+# The .tpinfo format
```

Comment:
Added a small readme, so that others may help writing exporters from other tools.